<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nikita Nemkin</author>
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Manager</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="tree_hosts">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_remove">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_refresh">
       <property name="text">
        <string>Re&amp;fresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_start_daemon">
       <property name="text">
        <string>&amp;Start Daemon</string>
       </property>
       <property name="icon">
        <iconset resource="../data/resources.qrc">
         <normaloff>:/icons/deluge/16x16/actions/system-run.png</normaloff>:/icons/deluge/16x16/actions/system-run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop_daemon">
       <property name="text">
        <string>&amp;Stop Daemon</string>
       </property>
       <property name="icon">
        <iconset resource="../data/resources.qrc">
         <normaloff>:/icons/deluge/16x16/actions/process-stop.png</normaloff>:/icons/deluge/16x16/actions/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="check_autoconnect">
        <property name="text">
         <string>Automatically connect to selected host on start-up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_do_not_show">
        <property name="text">
         <string>Do not show this dialog on start-up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_autostart">
        <property name="text">
         <string>Automatically start localhost if needed</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>check_autoconnect</zorder>
     <zorder>check_autostart</zorder>
     <zorder>check_do_not_show</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/resources.qrc"/>
  <include location="../data/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
